src/server/pachyderm_test.go

```diff
@@ -4538,6 +4538,9 @@ func TestDatumStatusRestart(t *testing.T) {
 		t.Skip("Skipping integration tests in short mode")
 	}
 
+	// TODO: Fix flaky test
```

Comment:
If you do `TODO(CORE-XXX)` that would be helpful for the future reader.  Could also go in the message and save people looking at CI results a trip to the code, actually.